src/store/validation/index.ts

```diff
@@ -151,10 +151,11 @@ export const actions: ActionTree<IValidationState, IRootState> = {
     Object.keys(backendErrorsList).forEach((currentUuid) => {
       const key = `backend/${type}/${currentUuid}`
       const currentErrors = backendErrorsList[currentUuid]
+      const uniqueErrors = getUniqueObjectValuesFromArray(currentErrors, 'message') as { message: string }[]
```

Comment:
Would it make sense to use `_.uniqBy` which seems to do the same thing?

Reply:
Exactly ! I've been looking for this yesterday, hahaha. 🤪 😝 😹